index.js

```diff
@@ -66,12 +73,35 @@ function concatEnvSchemas(schema, otherSchema) {
   }
 }
 
+function getCustomHeaders(headersKeyToProxy, headers) {
+  const headersKeys = headersKeyToProxy.split(',').filter(header => header)
```

Comment:
This can be cached. This line is executed for every `getDirectlyServiceBuilder` and `getServiceBuilder`.